espree.js

```diff
@@ -101,9 +166,9 @@ const parsers = {
 /**
  * Tokenizes the given code.
  * @param {string} code The code to tokenize.
- * @param {Object} options Options defining how to tokenize.
- * @returns {Token[]} An array of tokens.
- * @throws {SyntaxError} If the input code is invalid.
+ * @param {ParserOptions} options Options defining how to tokenize.
+ * @returns {acorn.Token[]|null} An array of tokens.
```

Comment:
`function tokenize` returns Espree tokens. The main difference is that in Espree tokens `type` is a string, while in Acorn tokens `type` is an object.

Also, I'm not sure if this function can return `null`?

Reply:
While `tokenize` could return `null`, indeed in this function since it forces `tokens: true`, indeed it should not. I temporarily forced it as an acorn token, but still need to investigate the Espree token type returning.

Reply:
Fixed now to indicate return of EspreeTokens.